reactioncommerce/reaction
# refactor: Update `inventoryQuantity` field to be `inventoryInStock`

Impact: **breaking|minor**  
Type: **refactor**

## Issue
A recent update to how we calculate inventory added a new field, `inventoryAvailalbeToSell`. This left us with two inventory fields, `inventoryAvailalbeToSell` and `inventoryQuantity`, in the `Products` collection.

In the `Catalog` collection, we renamed `inventoryQuantity` to `inventoryInStock`, as it was a better description of what the field actually is.

We initially left the `Products` collection alone, but feel like we should keep these field names in sync to avoid confusion.

## Solution
Rename the field `inventoryQuantity` to `inventoryInStock` in the `Products` collection.

## Breaking changes
No breaking changes in Reaction code. If you use the `inventoryQuantity` field in any custom code, it will need to be updated.

## Testing
1. Create orders with various amounts of products, and see that inventory correctly increments / decrements
1. See that Sold Out, Low Inventory badges work as they were
